.gitignore

```diff
@@ -160,7 +160,7 @@ Backup*/
 UpgradeLog*.XML
 
 # NuGet
-packages
+# packages
```

Comment:
Wow!  That would be a problem

Reply:
yeah, it was also causing vscode not to search inside that folder